Sources/Tracing/TracingTime.swift

```diff
@@ -23,10 +23,16 @@ import Glibc
 import Android
 #elseif canImport(Musl)
 import Musl
+#elseif canImport(WASILibc)
+    import WASILibc
 #else
 #error("Unsupported runtime")
 #endif
 
+#if canImport(_CWASI)
```

Comment:
```suggestion
#if os(WASI)
```

Reply:
@MaxDesiatov How about I do `#if os(WASI) && canImport(_CWASI)`? In theory all three options would have the same effect. Just want to preserve the canImport since it is conditionally available.